.github/workflows/build_and_test_release_latest.yml

```diff
@@ -60,25 +34,24 @@ jobs:
     # Needed for the CMakeLists.txt setup
     - run: export ROS_DISTRO=${{ matrix.ros_distro }}
     # Needed for the CMakeLists.txt setup
-    - run: export ROS_VERSION=1
+    - run: export ROS_VERSION=${{ matrix.ros_version }}
     - uses: ros-tooling/action-ros-ci@0.0.17
       with:
         source-ros-binary-installation: ${{ matrix.ros_distro }}
         package-name: aws_common
-        extra-cmake-args: --cmake-target tests
-        # schedule only works for the master branch, so specify release-latest
+        # schedule runs against the default branch (master), so specify release-latest via repos file
```

Comment:
Why was `extra-cmake-args` removed?

Reply:
Accident! I meant to work through making sure everything was passing before requesting review. It's been re-added